src/module.ts

```diff
@@ -0,0 +1,157 @@
+import { PanelConfig } from './panel-config';
+import { PanelCtrl, loadPluginCss } from 'grafana/app/plugins/sdk';
+import * as _ from 'lodash';
+import * as go from '../node_modules/gojs/release/go';
+import * as axios from '../node_modules/axios/dist/axios';
+
+class Ctrl extends PanelCtrl {
+  static templateUrl = "partials/template.html";
+
+  private panelDefaults = {
+    host: "localhost",
+    port: "7890",
+    application: "Servers,1.0.0",
+    user: "admin",
+    password: "admin"
+  };
+  private myFullDiagram: any = {
```

Comment:
Do not use any!
Gojs has types such as Diagram...

Reply:
done